go.mod

```diff
@@ -4,6 +4,7 @@ go 1.12
 
 require (
 	github.com/iancoleman/orderedmap v0.1.0
+	github.com/stretchr/testify v1.7.0
```

Comment:
I figure I'd add testify since that's what we all know and are familiar with in Gonfalon. But if we feel that it's too heavy we can swap to something lighter or just use the std test lib.